### DISCUSSION TITLE
Manual Installation with Next.js 14 App Router

### DISCUSSION THREAD

**Discussion by emtl688** [rainbow-me/rainbowkit#2047] [Feedback]
When was the last time the manual installation docs have been updated?

If I use create-next-app and then decide to install RainbowKit separately, following the manual install doesnt seem to work and is not adapted to the base layout for a fresh next application

**Comment by DanielSinclair** [comment]
@emtl688 I think you're describing the difference between App Router and Pages Router. Feel free to check out our examples here for the best configuration for both:
- App Router: https://github.com/rainbow-me/rainbowkit/tree/main/examples/with-next-app
- Pages Router: https://github.com/rainbow-me/rainbowkit/tree/main/examples/with-next
